clients/osv/src/funTest/kotlin/OsvServiceWrapperFunTest.kt

```diff
@@ -49,12 +49,14 @@ private val VULNERABILITY_FOR_PACKAGE_BY_INVALID_COMMIT_REQUEST = Vulnerabilitie
     commit = "6879efc2c1596d11a6a6ad296f80063b558d5e0c"
 )
 
+private val EMPTY_JSON_OBJECT = JsonObject(emptyMap())
```

Comment:
Nit: Should this go before `VULNERABILITY_FOR_PACKAGE_BY_COMMIT_REQUEST` for alphabetical sorting of private constants?

Reply:
Oh, I just realized constant are not strictly sorted alphabetically anyway, as `VULNERABILITY_FOR_PACKAGE_BY_INVALID_COMMIT_REQUEST` would also need to come before `VULNERABILITY_FOR_PACKAGE_BY_NAME_AND_VERSION`. So never mind.